docs/Target-devices-and-platforms.md

```diff
@@ -0,0 +1,46 @@
+# Currently working devices:
+Currently Servonk is targetting Gonk from B2G derived from Android Lollipop. There's also the desktop build with Glutin. 
+
+Using Gonk-L:
+* Sony Xperia Z3 Compact
+* LG Google Nexus 5
+
+Using Glutin:
+* Linux desktop Glutin
+
+To do: add links to the build instructions for each of the working devices.
+
+# Work in progress
```

Comment:
Not sure we need an empty section here?

Reply:
So if you have something to announce...? ;)